crates/swift-bridge-build/tests/sample-project/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+export SWIFT_BRIDGE_OUT_DIR="$(pwd)/generated"
```

Comment:
Need to use the same `THISDIR` trick that we use elsewhere. Otherwise if you run this from another directory you'll run into trouble.

i.e. if you did `./crates/swift-bridge-build/tests/sample-project/build.sh` you would end up creating `./generated` instead of `./crates/swift-bridge-build/tests/sample-project/generated`